src/referentielijsten/conf/base.py

```diff
@@ -1,382 +1,42 @@
-import os
-
 from django.urls import reverse_lazy
 
 from import_export.formats.base_formats import DEFAULT_FORMATS
+from notifications_api_common.settings import *  # noqa
+from open_api_framework.conf.base import *  # noqa
 
 from .api import *  # noqa
-from .utils import config
-
-# Build paths inside the project, so further paths can be defined relative to
-# the code root.
-
-DJANGO_PROJECT_DIR = os.path.abspath(
-    os.path.join(os.path.dirname(__file__), os.path.pardir)
-)
-
-BASE_DIR = os.path.abspath(
-    os.path.join(DJANGO_PROJECT_DIR, os.path.pardir, os.path.pardir)
-)
 
 #
 # Core Django settings
 #
-# SITE_ID = config("SITE_ID", default=1)
-
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config("SECRET_KEY")
-
-# NEVER run with DEBUG=True in production-like environments
-DEBUG = config("DEBUG", default=False)
-
-# = domains we're running on
-ALLOWED_HOSTS = config("ALLOWED_HOSTS", default="", split=True)
-USE_X_FORWARDED_HOST = config("USE_X_FORWARDED_HOST", default=False)
-
-IS_HTTPS = config("IS_HTTPS", default=not DEBUG)
-
-# Internationalization
-# https://docs.djangoproject.com/en/2.0/topics/i18n/
-
-LANGUAGE_CODE = "nl"
-
-TIME_ZONE = "UTC"
-
-USE_I18N = True
-
-USE_L10N = True
-
-USE_TZ = True
-
-USE_THOUSAND_SEPARATOR = True
-
-#
-# DATABASE and CACHING setup
-#
-DATABASES = {
-    "default": {
-        "ENGINE": config("DB_ENGINE", "django.db.backends.postgresql"),
-        "NAME": config("DB_NAME", "referentielijsten"),
-        "USER": config("DB_USER", "referentielijsten"),
-        "PASSWORD": config("DB_PASSWORD", "referentielijsten"),
-        "HOST": config("DB_HOST", "localhost"),
-        "PORT": config("DB_PORT", 5432),
-    }
-}
-
-DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
-
-CACHES = {
-    "default": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_DEFAULT', 'localhost:6379/0')}",
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            "IGNORE_EXCEPTIONS": True,
-        },
-    },
-    "axes": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_AXES', 'localhost:6379/0')}",
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            "IGNORE_EXCEPTIONS": True,
-        },
-    },
-}
-
 
 #
 # APPLICATIONS enabled for this project
 #
-
-INSTALLED_APPS = [
-    "django.contrib.auth",
-    "django.contrib.sessions",
-    "django.contrib.contenttypes",
-    # NOTE: If enabled, at least one Site object is required and
-    # uncomment SITE_ID above.
-    # 'django.contrib.sites',
```

Comment:
Hmm good point, I'm actually considering removing it from the installed apps here, because we want to move away from django.contrib.sites, it's kind of a relic of the past. I'll remove it here